operator/controllers/actionsrunner/util/converters.go

```diff
@@ -268,36 +268,9 @@ func ToJob(actionsRunner *inlocov1alpha1.ActionsRunner, actionsRunnerJob *inloco
 			BackoffLimit:          pointer.Int32Ptr(0),
 			Template: corev1.PodTemplateSpec{
 				Spec: corev1.PodSpec{
-					Volumes: []corev1.Volume{
-						corev1.Volume{
-							Name: "config-map",
-							VolumeSource: corev1.VolumeSource{
-								ConfigMap: &corev1.ConfigMapVolumeSource{
-									LocalObjectReference: corev1.LocalObjectReference{
-										Name: actionsRunner.GetName(),
-									},
-								},
-							},
-						},
-						corev1.Volume{
-							Name: "secret",
-							VolumeSource: corev1.VolumeSource{
-								Secret: &corev1.SecretVolumeSource{
-									SecretName: actionsRunner.GetName(),
-								},
-							},
-						},
-						corev1.Volume{
-							Name: "persistent-volume-claim",
-							VolumeSource: corev1.VolumeSource{
-								PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
-									ClaimName: actionsRunner.GetName(),
-								},
-							},
-						},
-					},
+					Volumes: withVolumes(actionsRunner),
 					Containers: []corev1.Container{
-						corev1.Container{
+						{
```

Comment:
```suggestion
						corev1.Container{
```

Reply:
Well, if we change it here, the other cases in the project would also need to be changed. Also, making the type explicit has worked better for me with the tooling I'm using.